prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,231 @@
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
+                        )
+                    )
+            else:
+                if not self.audit_resources or (
+                    is_resource_filtered(organization_arn, self.audit_resources)
+                ):
+                    self.organizations.append(
+                        Organization(
+                            arn=organization_arn,
+                            id=organization_id,
+                            status="ACTIVE",
+                            master_id=organization_master_id,
+                            policies=organization_policies,
+                            delegated_administrators=organization_d_administrator,
+                        )
+                    )
+                else:
+                    # is filtered
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
+                        )
+                    )
+
+        except Exception as error:
+            logger.error(
+                f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    # This method only returns "Attached directly" policies, missing "Inherited from..."
+    # def __list_policies_for_target__(self, target, enabled_policy_types):
+    #     logger.info("Organizations - List policies for target...")
+
+    #     try:
+    #         policies = []
+    #         list_policies_for_target_paginator = self.client.get_paginator(
+    #             "list_policies_for_target"
+    #         )
+    #         for policy_type in enabled_policy_types:
+    #             logger.info(
+    #                 "Organizations - List policies for target... - Type: %s",
+    #                 policy_type["Type"],
+    #             )
+    #             for page in list_policies_for_target_paginator.paginate(
+    #                 TargetId=target, Filter=policy_type["Type"]
+    #             ):
+    #                 for policy in page["Policies"]:
+    #                     policy_content = self.__describe_policy__(policy["Id"])
+    #                     policies.append(
+    #                         Policy(
+    #                             arn=policy["Arn"],
+    #                             id=policy["Id"],
+    #                             type=policy["Type"],
+    #                             aws_managed=policy["AwsManaged"],
+    #                             content=policy_content,
+    #                         )
+    #                     )
+    #     except Exception as error:
+    #         logger.error(
+    #             f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+    #         )
+    #     finally:
+    #         return policies
```

Comment:
Do we need this? We try to remove the commented code if we are not going to use it.

Reply:
We should be using this API Call (`list_policies_for_target`), but for some AWS Magic reason, it is not answering what it is supposed to answer; that's why I left it for reference in case someone wants to replace `list_policies` with this one.

Reply:
Ok, we can leave this there for the future.